bin/build

```diff
@@ -13,6 +13,6 @@ BUILD_SCRIPT="cd /src && "\
 "rm -rf build && mkdir build && "\
 "go build -o build/codeclimate-gofmt ."
 
-docker run -v "$SRC_ROOT:/src" yunspace/alpine-golang /bin/sh -c "$BUILD_SCRIPT"
+docker run -v "$SRC_ROOT:/src" golang:latest /bin/sh -c "$BUILD_SCRIPT"
```

Comment:
note that this script is using the full golang image, but the dockerfile has the `golang:alpine` image. (The full image seems to have git support for grabbing packages)